server/src/main/java/server/logger/BazelServerLogger.java

```diff
@@ -0,0 +1,28 @@
+package server.logger;
+
+
+import java.io.PrintStream;
+
+public class BazelServerLogger {
```

Comment:
Maybe we ought to call this class `Logger` instead of `BazelServerLogger`? This logger will only be used in the context of the server, so the `BazelServer` prefix feel redundant